web/app_dev.php

```diff
@@ -19,7 +19,7 @@
     header('HTTP/1.0 403 Forbidden');
     exit('You are not allowed to access this file. Check '.basename(__FILE__).' for more information.');
 }
-//*/
+*/
```

Comment:
Protip: this modification is useless. When you remove the first `/` it automatically comments the whole block and when you add it, it uncomments the block. :)